lib/livebook_web/components/core_components.ex

```diff
@@ -444,7 +444,7 @@ defmodule LivebookWeb.CoreComponents do
         <%= @label %>
       </span>
       <span class={
-        "text-gray-800 text-sm font-semibold #{if @one_line, do: "whitespace-nowrap overflow-auto tiny-scrollbar"}"
+        "text-gray-800 text-sm font-semibold #{if @one_line, do: "overflow-auto tiny-scrollbar"}"
```

Comment:
We need `whitespace-nowrap` here, the `:one_line` option is to enforce the text is a single line with scrollbar if too long. You can just omit the `one_line` option when using `<.labelled_text>` :)